Samurai/Modules/TippingModule.cs

```diff
@@ -88,7 +88,7 @@ await ReplyAsync
         {
             if (totalAmount == decimal.MinusOne)
             {
-                totalAmount = (decimal)Math.Pow(10, Random.NextDouble()) - 1;
+                totalAmount = (decimal)Math.Pow(10, Program.Random.NextDouble()) - 1;
```

Comment:
It seems to be verbose expressions. If you are feeling risks about it, please make unsafe member names safety.